Zappy_IA_Report/DTC/Video/Tutorial3.py

```
@@ -0,0 +1,17 @@
+import pandas as pd
+
+d = {'sales': [100000, 222000, 1000000, 522000, 111111],
+    'city': ['Tampa', 'Tampa', 'Orlando', 'Jacksonville', 'Miami'],
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)